app/forms.py

```diff
@@ -0,0 +1,96 @@
+# import re
+from django import forms
+# from PIL import Image
+from django.contrib.auth.models import User
+from django.utils.translation import ugettext_lazy as _
+# from django.core.exceptions import ValidationError
+import datetime
+from django.forms.extras.widgets import SelectDateWidget
+
+
+class RegistrationForm(forms.Form):
+    this_year = datetime.date.today().year
```

Comment:
Simon, we believe that we can get the current year from the datetime library once if we execute the view layer in the app like hitting the URL in the browser. We are not sure about your comment over restart. Please advice on the same.